src/Libraries/Microsoft.Extensions.AI.Abstractions/Contents/DataContent.cs

```diff
@@ -105,6 +105,14 @@ public DataContent(ReadOnlyMemory<byte> data, string? mediaType = null)
         _data = data;
     }
 
+    /// <summary>
+    /// Determines whether the <see cref="MediaType"/> has the specified prefix.
+    /// </summary>
+    /// <param name="prefix">The media type prefix.</param>
+    /// <returns><see langword="true"/> if the <see cref="MediaType"/> has the specified prefix, otherwise <see langword="false"/>.</returns>
+    public bool HasMediaTypePrefix(string prefix)
```

Comment:
Reading this again, simply "MediaTypeStartsWith" instead of "HasMediaTypePrefix" might be a better name.